### DISCUSSION TITLE
Tab Modifier reborn & v1.0

### DISCUSSION THREAD

**Discussion by sylouuu** [furybee/chrome-tab-modifier#271] [Announcements]
Dear Tab Modifier lovers,

We are thrilled to announce that the Tab Modifier project is active again!

This revival has been made possible thanks to the significant contributions of @sebastienfontaine, who has modernized the entire codebase. Most of the original features have been preserved, and we were encouraged to expedite our efforts due to Google's deprecation of Manifest v2 in June.

To formalize this collaboration, we have decided to move the repository from my personal GitHub account to the "furybee" organization.

More features are on the way! As of this writing, 80,000 users have the extension installed.

Check out the [1.0.0 release](https://github.com/furybee/chrome-tab-modifier/releases/tag/1.0.0)!

Stay tuned for more updates!

Syl

**Comment by paul1149** [comment]
Excellent news for all of us who want to take control of our Web experience as much as possible. THANK YOU!

If I may suggest, before upgrading, Export the old TM's rules to a backup file, so that they can be easily imported after the upgrade, as the rules do not on their own survive the upgrade.

Love the Dark Mode!

**Comment by ciltocruz** [comment]
Thanks for the update but... Where are my rules? They have all disappeared!

**Comment by sebastienfontaine** [comment]
Thanks for letting me know about the issue with your rules disappearing after the update. I'm really sorry about that! 😔

I've fixed the problem, but it might take a little while for Google to review and approve the update. Thanks for your patience while we get this sorted.

If you have any questions or need help in the meantime, just give me a shout.

**Comment by gitsang** [comment]
Already updated, can the lost rules be recovered?

**Comment by Atilil** [comment]
Title matcher and URL matcher don't work anymore, tab name are just replace by @1 or $1
![image](https://github.com/furybee/chrome-tab-modifier/assets/155106095/b9f23c88-303b-424d-a340-b53cea6d9779)

**Reply by klhoste** [reply]
I'm with 1.0.2 version now and got kind of same issue.

URL Fragment: `(?<!dev).test.com`
Tab title: `[$1] {title}`
URL Matcher: `//(.*)\.test.com`

When I go to foo.test.com, with previous version tab title was "[foo] My title", but now it is "[$1] My title".

**Comment by klhoste** [comment]
Bonjour,

Thanks for maintaining this extension.
My chrome was updated with the latest extension version, but all my rules disappeard, how to recover them ?

Edit: I just see your last message about a fix, I'm in version 1.0.1, I'll wait for the next update... 

Edit bis: rules came back with 1.0.2 👍 but i got the same issue as Atilil. 

Kévin

**Reply by ciltocruz** [reply]
Same for me. No old rules recovered after update to 1.0.2

**Comment by sylouuu** [comment]
Please use the [Issues tab](https://github.com/furybee/chrome-tab-modifier/issues) to report bugs, not this discussion topic.

If your rules are not recovered with `1.0.2`, recreate them and backup the JSON file (always a good idea).
The `1.0.3` is still not validated by Google yet.

Check out the [releases page](https://github.com/furybee/chrome-tab-modifier/releases) for changelog.

Bests

**Comment by ciltocruz** [comment]
Hello. Rules from before version 1.0 are still not available. I have lost more than 30 rules that I had before the update. How can we recover them?

I am not opening an issue because this topic is being discussed here.

**Reply by sylouuu** [reply]
https://github.com/furybee/chrome-tab-modifier/discussions/271#discussioncomment-9711614

**Reply by ciltocruz** [reply]
haha, recreate? manually? 😑😑😶😶

**Reply by sylouuu** [reply]
We have no servers, so recreate once again and export your rules. Be kind, it's free.
